tests/accessor_legacy/accessor_api_common_all.h

```diff
@@ -61,7 +61,8 @@ void check_accessor_members(sycl_cts::util::logger &log,
                 "value_type is of wrong type");
 
   using reference = typename acc_t::reference;
-  static_assert(std::is_same_v<reference, T &>, "reference is of wrong type");
+  static_assert(std::is_same_v<reference, value_type &>,
```

Comment:
I have already seen this. :-)

Reply:
Right. I cherry-picked https://github.com/KhronosGroup/SYCL-CTS/pull/504/commits/650660837c346d2edbddec7a4ba9c02109898c2d from https://github.com/KhronosGroup/SYCL-CTS/pull/501. 

As @steffenlarsen said in [this comment](https://github.com/KhronosGroup/SYCL-CTS/pull/501#issuecomment-1418998838), this change should go with or after DPC++ version update. CI says they go together. SYCL implementation version update in CI can validated only with PR created from the local branch. #501 uses private fork, so I created this PR to test DPC++ version uplift. 

NOTE: the PR is a draft and not ready for review yet. I need to make sure there are not regressions added with the version uplift.

Reply:
Thanks @bader ! I have closed the other PR as we cannot go ahead with it without the bump anyway. Open discussions can continue on there, but no point in keeping it open.